### DISCUSSION TITLE
Going crazy trying to get 2.1.0 running using Docker Compose.

### DISCUSSION THREAD

**Discussion by ziggekatten** [apache/superset#24212] [Q&A / Help]
So, I've been running versions 1.x to 2.0.1 without any issues using the supplied composefiles.

Server: Ubuntu 22.04

version 2.0.1 in directory /opt/superset-2.0.1
version 2.1.0 in directory /opt/superset-2.1.0

Doing an docker-compose -f docker-compose-non-dev.yaml up in the 2.0.1 directory works like an charm. Shutting it down and instead change to the 2.0.1 directory an do the same, it pulls the apache/superset:2.1.0 image, starts the database and redis, but fails when trying to start app/worker/beat with errors. Repulling the image does not help, and I have tried an image that works in our prod environment. Scratching my head here, as I think it is an Docker/OS issue. Errors below:

-------------------------------------------------------------
`Creating superset_app         ... error
Creating superset_init        ...

Creating superset_worker      ... error

ERROR: for superset_app  Cannot start service superset: unable to find user root: no matching entries in passwd file

Creating superset_init        ... error

ERROR: for superset_init  Cannot start service superset-init: unable to find user root: no matching entries in passwd file

ERROR: for superset-worker-beat  Cannot start service superset-worker-beat: unable to find user root: no matching entries in passwd file

ERROR: for superset  Cannot start service superset: unable to find user root: no matching entries in passwd file

ERROR: for superset-worker  Cannot start service superset-worker: unable to find user root: no matching entries in passwd file

ERROR: for superset-init  Cannot start service superset-init: unable to find user root: no matching entries in passwd file
`
----------------------------------------

Trying to getting the image to spin up an container with: docker run -it apache/superset:2.1.0 /bin/bash

Doing this with 2.0.1 works fine, but with 2.1.0 on the same machine yields another issue that I have never seen before:

---------------------------------------------------------------------------------------------------------------------------------------------------
`docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/bin/bash": stat /bin/bash: no such file or directory: unknown.
ERRO[0000] error waiting for container:`

**Comment by rusackas** [comment]
Closing this since we no longer support 2.x (or 3.x). Come on in and try 4.x while it lasts! 5.x is coming soon!
